docker/Dockerfile

```diff
@@ -0,0 +1,47 @@
+ARG BASE=21.07
+FROM nvcr.io/nvidia/pytorch:${BASE}-py3 as base
+
+FROM base as trtorch-builder-base
+
+RUN apt-get update && apt-get install -y curl gnupg
+RUN curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > /etc/apt/trusted.gpg.d/bazel.gpg
+RUN echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list
+
+RUN apt-get update && apt-get install -y bazel-4.0.0
+RUN ln -s /usr/bin/bazel-4.0.0 /usr/bin/bazel
+
+# Workaround for bazel expecting both static and shared versions, we only use shared libraries inside container
+RUN cp /usr/lib/x86_64-linux-gnu/libnvinfer.so /usr/lib/x86_64-linux-gnu/libnvinfer_static.a
+
+RUN apt-get update && apt-get install -y locales ninja-build && rm -rf /var/lib/apt/lists/* && locale-gen en_US.UTF-8
+
+FROM trtorch-builder-base as trtorch-builder
+
+COPY . /workspace/trtorch/src
+WORKDIR /workspace/trtorch/src
+RUN cp ./docker/WORKSPACE.cu.docker  WORKSPACE
+
+# This script builds both libtrtorch bin/lib/include tarball and the Pythin wheel, in dist/  
+RUN ./docker/dist-build.sh
+
+# This currently does not work with BUILDKIT=1
+# RUN ./docker/dist-test.sh
```

Comment:
Is the intention here only to run python tests ? If so, why ? 
Besides, I'm not sure if this would work. I used to run python tests directly. @narendasan  can comment more on this.

Reply:
Yeah the bazel based python tests usually hit errors related to the bazel sandbox. we usually use pytest directly.

Reply:
It won't work, I will clean those up.